pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/docker/logrotate/logrotate.go

```diff
@@ -16,52 +16,59 @@ package logrotate
 
 import (
 	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
-
 	"k8s.io/utils/pointer"
 )
 
 // Config returns the content for logrotate units and files.
 // Whenever logrotate is ran, this config will:
-//   - rotate the log file if its size is > 100Mb OR if one day has elapsed
+//   - rotate the log file if its size is > 100Mb (only in docker case) OR if one day has elapsed
 //   - save rotated logs into a gzipped timestamped backup
 //   - log file timestamp (controlled by 'dateformat') includes seconds too. This
 //     ensures that logrotate can generate unique logfiles during each rotation
 //     (otherwise it skips rotation if 'maxsize' is reached multiple times in a
 //     day).
 //   - keep only 14 old (rotated) logs, and will discard older logs.
+//
+// prefix carries the target container runtime such as (containerd, docker)
 func Config(pathConfig, pathLogFiles, prefix string) ([]extensionsv1alpha1.Unit, []extensionsv1alpha1.File) {
```

Comment:
Same as in the containerd file, prefix can be now dropped.

Reply:
In my opinion the original design helps readability if you work on the host for whatever reason. Why change it?